modules/nw-ingress-controller-configuration-parameters.adoc

```diff
@@ -159,6 +159,15 @@ These adjustments are only applied to cleartext, edge-terminated, and re-encrypt
 
 For request headers, these adjustments are applied only for routes that have the `haproxy.router.openshift.io/h1-adjust-case=true` annotation. For response headers, these adjustments are applied to all HTTP responses. If this field is empty, no request headers are adjusted.
 
+|`httpCompression`
+|`httpCompression` defines the policy for HTTP traffic compression. The optional `ROUTER_ENABLE_COMPRESSION` field turns on compression for the MIME types, which are specified in the `mimeTypes` variable. Compression is not enabled by default.
+
+You can configure compression of MIME types using the environment variables:
+
+* `ROUTER_ENABLE_COMPRESSION` compresses the body of the response when set to `true` or `TRUE` whenever possible.
+
+* `mimeTypes` defines a list of MIME types to which compression is applied.  The types are: application, image, message, multipart, text, video, or a custom type prefaced by `X-`. To specify a format, use the type/subtype [;attribute=value]. For example, `text/css; charset=utf-8`, `text/html`, `text/*`, `image/svg+xml`, `application/octet-stream`, `X-custom/customsub`.
+
```

Comment:
There is a useful `note` in https://github.com/openshift/api/blob/master/operator/v1/types_ingress.go#L257-L261

It also might not be a bad idea to reference the required formatting of a single mimeType, see https://github.com/openshift/api/blob/master/operator/v1/types_ingress.go#L271-L284

Reply:
Ok I added the same link from the Configuring router compression topic.